base/src/test/java/uk/ac/ebi/atlas/utils/EuropePmcClientIT.java

```diff
@@ -25,27 +23,33 @@
     public void publicationForValidDoi() {
         Optional<Publication> result = subject.getPublicationByIdentifier("10.1126/sciimmunol.aan8664");
 
-        assertThat(result.isPresent(), is(true));
-        assertThat(result.get().getDoi(), is("10.1126/sciimmunol.aan8664"));
-        assertThat(result.get().getAuthors(), not(isEmptyString()));
-        assertThat(result.get().getTitle(), not(isEmptyString()));
+        assertThat(result.isPresent()).isTrue();
+
+        assertThat(result.orElseThrow(RuntimeException::new))
+                .extracting("doi", "authors", "title")
+                .isNotEmpty();
+
+        assertThat(result.orElseThrow(RuntimeException::new).getDoi()).isEqualToIgnoringCase("10.1126/sciimmunol.aan8664");
```

Comment:
You can conflate the two tests:
```
        assertThat(result.orElseThrow(RuntimeException::new))
                .hasFieldOrPropertyWithValue("doi", "10.1126/sciimmunol.aan8664")
                .extracting("doi", "authors", "title")
                .isNotEmpty();
```

You don’t get the benefit of case insensitive match, though.